src/storage/src/monitor/monitored_store.rs

```diff
@@ -236,6 +236,16 @@ impl MonitoredStateStore<HummockStorage> {
     }
 }
 
+impl MonitoredStateStore<HummockStorageV1> {
+    pub fn sstable_store(&self) -> SstableStoreRef {
```

Comment:
Do we still need this two methods after we introduce the `HummockTrait` for StateStoreImpl?